src/apps/user/models.py

```diff
@@ -37,14 +41,23 @@ class Meta:
 
 
 class UserProfile(models.Model):
-    profile_picture = models.ImageField(upload_to="profile_pictures/", blank=True, null=True)
+    @staticmethod
+    def profile_picture_path(instance, filename):
```

Comment:
move to storages file
treat is util function that helps with upload_to paths

later we can reuse it for other private stuff as user review images